src/app/my-account/my-account/my-account.component.ts

```diff
@@ -44,7 +53,34 @@ export class MyAccountComponent implements OnInit {
     dialogRef.componentInstance.onClosed.pipe(take(1)).subscribe((confirmed: boolean) => {
       dialogRef.close();
       if (confirmed) {
-        // TODO: call to api
+        if (announcement.type === TransportOffer.TypeEnum.Transport) {
+          this.transportResourceService
+            .deleteTransport(announcement.id!)
+            .pipe(
+              switchMap((_) => {
+                return this.myOffersResource.listMyOffers(this.pageRequest);
+              })
+            )
+            .subscribe((data) => (this.myAnnouncements = data));
```

Comment:
I'm not going to block over this but I dislike such syntax, it's using a statement as an expression and it evaluates to that parameter which in some context can bite where a non-undefined return value may have some effect. This is much clearer, even if it takes more lines:
```suggestion
            .subscribe((data) => {
              this.myAnnouncements = data;
            }));
```

The same below.